storybook/src/components/Select/Select.stories.tsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { Select } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Components/Forms/Select',
+  component: Select,
+  args: {
+    invalid: false,
+    required: false,
+    disabled: false,
+    multiple: false,
+    children: [
+      <Select.Option key="0" disabled selected hidden>
```

Comment:
[This article](https://www.w3docs.com/snippets/css/how-to-create-a-placeholder-for-an-html5-select-box-by-using-only-html-and-css.html) has a cool way to create a placeholder for the select and I decided to adopt it. What do you guys think? Shall I document this?

Reply:
Does not seem to hide on iOs though:
<img width="2037" alt="image" src="https://github.com/Amsterdam/design-system/assets/474946/666f0d3c-ad48-4272-954a-4b7bfd7a4556">

Reply:
If it causes issues, I wouldn't use it